docs/guide/tutorial-3-appearance-15-minutes.md

```diff
@@ -46,25 +42,19 @@ If the DEF-USE mechanism of the previous tutorial has been correctly implemented
 The aim of this subsection is to apply a texture on the ball.
 A texture on a rolling object can help to appreciate its movement.
 
-> **Hands on**: Similarly add a [PBRAppearance](../reference/pbrappearance.md) node to the ball. As before, set the `metalness` field to 0 and the `roughness` field to 1. Add an [ImageTexture](../reference/imagetexture.md) node to the `baseColorMap` field of the [PBRAppearance](../reference/pbrappearance.md) node.
+> **Hands on #3**: Similarly add a [PBRAppearance](../reference/pbrappearance.md) node to the ball. As before, set the `metalness` field to 0 and the `roughness` field to 1. Add an [ImageTexture](../reference/imagetexture.md) node to the `baseColorMap` field of the [PBRAppearance](../reference/pbrappearance.md) node.
```

Comment:
Here we should clearly use a pre-defined appearance (but this change might be done in a second pass on the tutorial).

Reply:
Yes, I prefer to do it later, as this PR was simply aiming at tutorial 1 and the rest is just homogenization.